apps/test/unit/templates/projects/ProjectAppTypeAreaTest.js

```diff
@@ -59,6 +59,27 @@ describe('ProjectAppTypeArea', () => {
       expect(stubAjax).not.to.have.been.called;
     });
 
+    it('renders a working link to view more projects of a specific type', () => {
+      var viewMoreLink = "more App Lab projects";
```

Comment:
Should this be a localized string instead of hard-coded English?  We're able to reference our i18n strings in tests.

Reply:
The strings aren't localized in other tests in this file, so I followed the pattern.  I'm just using it to check that whatever string was passed as the `labViewMoreString` prop is what ends up in the text of the link.